test/03LessonThreeTest.js

```diff
@@ -5,33 +5,42 @@ var assert = require('assert');
 
 function buildArray(first, second, third) {
     // return array with the items first, second, and third in an array.
-
+    var array = [first, second, third]
+    return array;
 }
 
 function returnThirdItem(arr) {
     // should return the third item in the array
-
+    return arr[2];
 }
 
 function setFirstItem(arr, newFirstItem) {
     // should set the first item in the array with newFirstItem
-
+    arr[0] = newFirstItem;
+    return arr;
 }
 
 function returnCenterItem(fiveByFiveArray) {
     // returns the "center" item in a 5 x 5 array
     // ex. 3 x 3 array [[1, 2, 3], [4, 5, 6], [7, 8, 9]] the center item is 5
-
+    var fiveByFiveArray = [[1, 2, 3, 4, 5],
```

Comment:
You shouldn't be defining this variable here. It's overwriting the function parameter of the same name.